### PR TITLE
adding packages from bioconductor

### PR DESCRIPTION
**Pull request by nrichart**

### CODE REVIEW

File: humagne.yaml [hunk 1]
```diff
@@ -476,6 +477,9 @@ packages:
       - lapack
     specs:
       - r+external-lapack ^freetype@2.7.1 ^harfbuzz@2.2.0
+      - r-limma ^r+external-lapack ^freetype@2.7.1 ^harfbuzz@2.2.0
```
Comment (nrichart) [inline humagne.yaml:480]:
```suggestion
      - r-limma ^r+external-lapack ^freetype@2.7.1\
```

**Review by alalazo — APPROVED** [review]
:wave: :slightly_smiling_face:

**Comment by alalazo** [comment]
Just trolling to say "hi" I somehow received a notification to review this right now.

**Comment by nrichart** [comment]
I used this PR to do a demonstration of the capabilites of PRs... I though I canceled my modifications :)
Hi :wave: :smile:

**Pull request closed without merging**